contrib/kml_export.lua

```diff
@@ -348,7 +365,7 @@ local function create_kml_file(storage, image_table, extra_data)
         else
             kmlFileOpenCommand = "xdg-open "..exportDirectory.."/\""..exportKMLFilename.."\""
 	end
-        dt.control.execute( kmlFileOpenCommand) 
+        coroutine.yield("RUN_COMMAND", kmlFileOpenCommand) 
```

Comment:
Last time I talked with @boucman the plan was to add something like the code below to darktable, to avoid that we have an if cause in every script. But I don't know the actual status.

local dt = require "darktable"
local yield_old = coroutine.yield
coroutine.yield = function(yield_type, command)
  if (dt.configuration.api_version_major < 4) then
    yield_old(yield_type,command)
  else
    if (yield_type == "RUN_COMMAND") then
      dt.control.execute(command)
    elseif (yield_type == "FILE_READABLE") then
      dt.control.read(command)
    elseif (yield_type == "WAIT_MS") then
      dt.control.sleep(command)
    end
  end
end